app/models/single_use_link.rb

```diff
@@ -2,18 +2,21 @@ class SingleUseLink < ActiveRecord::Base
   validate :expiration_date_cannot_be_in_the_past
   validate :cannot_be_destroyed
 
+  alias_attribute :downloadKey, :download_key
```

Comment:
Should these alias methods be deprecated so as to discourage their use?

Reply:
I wanted to deprecate them, but I couldn't figure out how to deprecate them comprehensively. They're not just methods; they're attributes that can show up anywhere. I don't know how to warn that, for example, `SingleUseLink.find_by(downloadKey: 'foo')` is deprecated in favor of `SingleUseLink.find_by(download_key: 'foo')`.

Reply:
I think at least deprecating the attributes would be better than not deprecating any usages at all.  I think doing a `SingleUseLink.find_by(downloadKey: 'foo')` is broken after this PR.  But I don't know how you would avoid that.

Reply:
The `alias_attribute` you highlighted specifically supports the continued use of the old attributes everywhere, even as hash keys in `find_by` and `create`. I can add deprecations to the straight-up attribute getter/setter methods, but not the other usages.